packages/types/src/constants/ForkliftControllerModel.v1beta1.ts

```diff
@@ -26,3 +26,6 @@ export const ForkliftControllerModelGroupVersionKind = {
   group: 'forklift.konveyor.io',
 };
 export const ForkliftControllerModelRef = 'forklift.konveyor.io~v1beta1~ForkliftController';
+
+// TODO: should api version contain also group?
```

Comment:
> should the api version contain the group
all this files are auto generated from the go type definitions so , whatever they say is true by definition :-)

p.s
this file is auto generated, I think it's safer to create new file for this const